<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/index/taskbar</normaloff>:/new/index/taskbar</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1060</width>
     <height>670</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1656544</width>
     <height>3565600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1050</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1050</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="infoWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
border-bottom:1px solid gray
</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>23</y>
         <width>130</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
border-image: url(:/image/title.png);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="freshButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>60</width>
         <height>15</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#freshButton {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/fresh_normal);
}
QPushButton#freshButton:hover {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/fresh_hover);
}
QPushButton#freshButton:pressed {
    border: 1px solid white;
    border-radius: 4px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/fresh_press);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>354</x>
         <y>30</y>
         <width>60</width>
         <height>15</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#updateButton {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/update_normal);
}
QPushButton#updateButton:hover {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/update_hover);
}
QPushButton#updateButton:pressed {
    border: 1px solid white;
    border-radius: 4px;
    color:white;
    font-weight:bold;
    border-image: url(:/new/index/update_press);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="helpButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>30</y>
         <width>30</width>
         <height>15</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#helpButton {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
	border-image: url(:/new/index/image/help_normal.png);
}
QPushButton#helpButton:hover {
    border: 1px solid white;
    border-radius: 1px;
    color:white;
    font-weight:bold;
	border-image: url(:/new/index/image/help_hover.png);
}
QPushButton#helpButton:pressed {
    border: 1px solid white;
    border-radius: 4px;
    color:white;
    font-weight:bold;
	border-image: url(:/new/index/image/help_hover.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>72</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/index/image/titlelogo.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1030</width>
        <height>500</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{       /*主界面*/
        border:none;
        background:white;
}
QTabWidget::tab-bar{        /*tab所在位置*/
        alignment:right;
      /* left:500px;*/
}
QTabBar::tab:first{
        border: 0px solid #C4C4C3;     /*边界宽度和颜色*/
        border-top-color:#212e31;       /*上边界颜色*/
	    border-image: url(:/new/index/image/view_normal.png);
        color:black;
        font-family:&quot;Nuosu SIL&quot;;
        font-size:11px;
        font-weight:bold;
		min-height:20 px;
        min-width:60 px;
}
QTabBar::tab:first:selected{
        border: 0px solid #C4C4C3;     /*边界宽度和颜色*/
        border-top-color:#212e31;       /*上边界颜色*/
        border-image: url(:/new/index/image/view_now.png);
        color:black;
        font-family:&quot;Nuosu SIL&quot;;
        font-size:11px;
        font-weight:bold;
		min-height:20 px;
        min-width:60 px;
}
QTabBar::tab:last{
        border: 0px solid #C4C4C3;     /*边界宽度和颜色*/
        border-top-color:#212e31;       /*上边界颜色*/
        border-image: url(:/new/index/image/list_normal.png);        
        color:black;
        font-family:&quot;Nuosu SIL&quot;;
        font-size:11px;
        font-weight:bold;
		min-height:20 px;
        min-width:60 px;
}
QTabBar::tab:last:selected{
        border: 0px solid #C4C4C3;     /*边界宽度和颜色*/
        border-top-color:#212e31;       /*上边界颜色*/
        border-image: url(:/new/index/image/list_now.png);
        color:black;
        font-family:&quot;Nuosu SIL&quot;;
        font-size:11px;
        font-weight:bold;
		min-height:20 px;
        min-width:60 px;
}

</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="viewTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1040</width>
             <height>500</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/new/index/image/background.png);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>177</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" name="vmsGridLayout"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>275</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>1050</width>
            <height>580</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>156422</width>
            <height>3543650</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none</string>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>171</number>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VID</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>在线状态</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>外部ID</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>外部端口</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建时间</string>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fresh_label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(144, 155, 165);</string>
         </property>
         <property name="text">
          <string>refeshing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="refresh_label">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="view_pushButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-image: url(:/new/index/image/view_normal.png);
}
QPushButton:hover {
border-image: url(:/new/index/image/view_now.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_pushButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-image: url(:/new/index/image/list_normal.png);
}
QPushButton:hover {
border-image: url(:/new/index/image/list_now.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
